test/unit/network_spec.js

```diff
@@ -161,4 +162,990 @@ describe('network', function() {
       done.fail(reason);
     });
   });
+
+  describe('contentDisposition', function() {
```

Comment:
Those tests are derived from https://github.com/jshttp/content-disposition/blob/82fd7db3aa663438166f970efb5e337fa63b4465/test/test.js. When you simplify the code you'll see that not all tests are relevant, so you can cut those as well.

Also, since a significant portion of this test is based on that content-disposition module, add a single line around here that links to the original source (with version/tag).